app/css/index.css

```diff
@@ -951,3 +955,44 @@ h3 {
     font-weight: 600;
     color: #f0f0f0;
 }
+
+.forkMarket {
+	border: 4px dashed var(--input-border);
+	border-radius: var(--btn-radius);
+	box-shadow: var(--btn-shadow);
+	padding: 10px;
+	position: relative;
+}
+
+.reportingRules {
+	border: 4px solid var(--input-border);
+	border-radius: var(--btn-radius);
+	box-shadow: var(--btn-shadow);
+	padding: 10px;
+}
+
+.border-text {
+	position: absolute;
+	top: -1.2em;
+	left: 50%;
+	transform: translateX(-50%);
+	background: #1e1e2f;
+	padding: 5px;
+	color: var(--input-border);
+	font-weight: bold;
+}
+
+.universe-forking {
+    background-color: red;
```

Comment:
Can we choose a color that isn't quite so painful to read text on?  Can still be red, but something a tad more tame.